.github/workflows/windows.yml

```diff
@@ -33,15 +34,19 @@ jobs:
         if: ${{ github.event_name == 'workflow_dispatch' && inputs.debug_enabled }}
         with:
           detached: true
+      - name: Calculate PACBOY_PACKAGES
+        shell: bash
+        run: |
+          pwd
+          ./scripts/msys2-install-dependencies.sh ${{ matrix.qt }}
       - name: Setup MSYS2 Environment
         uses: msys2/setup-msys2@v2
+        id: msys2
         with:
           msystem: UCRT64
           update: true
-      - name: Install MSYS2 Dependencies
-        run: |
-          pwd
-          ./scripts/msys2-install-dependencies.sh ${{ matrix.qt }}
+          install: libxml2
```

Comment:
Do you know if/why libxml2 needs to be installed like this instead of libxml2:p via pacboy?

Reply:
Not a clue. I just copied what the script does.